src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -108,8 +118,8 @@ public static TransactionViewModel validate(final byte[] bytes, int minWeightMag
 
     private final AtomicInteger nextSubSolidGroup = new AtomicInteger(1);
 
-    public boolean checkSolidity(Hash hash, boolean milestone) throws Exception {
-        if(TransactionViewModel.fromHash(tangle, hash).isSolid()) {
+    public boolean checkSolidity(Hash hash, boolean priority) throws Exception {
+        if(TransactionViewModel.fromHash(tangle, hash).subtangleStatus() == SubtangleStatus.SOLID) {
             return true;
         }
         Set<Hash> analyzedHashes = new HashSet<>(Collections.singleton(Hash.NULL_HASH));
```

Comment:
`boolean solid = true;` not used anymore
(line below - couldn't add comment to that line)